edx_lint/files/pylintrc

```diff
@@ -36,28 +249,88 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
+    bad-continuation,
+    invalid-name,
+    misplaced-comparison-constant,
+    file-ignored,
+    too-many-nested-blocks,
+    too-many-statements,
+    too-many-boolean-expressions,
+    protected-access,
```

Comment:
I'm not sure pylint will let us be that specific. And per my comment, this is something that I'd like for reviewers make the call on. That said, we could leave it on, but just give blanket permission to allow `_foo` access in test files (using a file-level `pylint: disable`).